File: twitch/cmd/fesses.py
"""Cmd fesses"""
async def fesses_cmd(self, ctx):
    """fonction de la cmd fesses"""
    if ctx.author.name in self.config['ADMIN'] or ctx.author.name in self.config['PRENIUM']:
        await ctx.send(
            "SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss " 
            "SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss " 
            "SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss " 
            "SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss "
            "SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss "
            "SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss SSSsss "
            "SSSsss SSSsss SSSsss SSSsss SSSsss"
        )
    else:
        await ctx.send("fesses")

File: src/cmd/fesses.ts
import tmi from 'tmi.js';
import type { Config } from '../types';

export const fesses_cmd = (BOT:tmi.Client, channel:string, author:string, config:Config) => {
    console.log(`CMD : ${author} a utilisé la cmd fesses`);
    if (config.ADMIN.includes(author) || config.PRENIUM.includes(author)) {
        BOT.say(channel, 'SSSsss '.repeat(71).trim());
    } else {
        BOT.say(channel, 'fesses');
    }
};